Assets/Editor/E05_Controls.cs

```diff
@@ -77,6 +101,9 @@ void AddTask()
             {
                 m_TasksContainer.contentView.AddChild(CreateTask(m_TextField.text));
                 m_TextField.text = "";
+
+                // Give focus back to text field.
+                m_TasksContainer.focusController.SwitchFocus(m_TextField);
```

Comment:
Right, I'm just wondering if would be easier to myVisualElement.TakeFocus(). But your approach is fine ; just debating if FocusController is a class meant to be public / documented etc.